core/src/mindustry/world/blocks/defense/turrets/Turret.java

```diff
@@ -397,10 +402,6 @@ public void updateTile(){
                     }
                 }
             }
-
-            if(coolant != null){
```

Comment:
Where did the coolant check go?

Reply:
`ReloadTurret#updateCooling` already checks for coolant